core-test/src/test/scala/org/bitcoins/core/protocol/dlc/ContractDescriptorTest.scala

```diff
@@ -118,9 +128,6 @@ class ContractDescriptorTest extends BitcoinSUnitTest {
     val oldHex =
       "fda720260002fda7261a0002010000000000000000000000010300000000000000640000fda724020000"
 
-    //i need to write a test case to make sure we can parse both the old serialization
-    //format and the new serialization format
-    //i also need to verify that the data structures above are correct in the diff
     val actual = ContractDescriptorV1TLV.fromHex(oldHex)
```

Comment:
are there any tests for anything other than `Post144Pre163`?

Reply:
The `val oldHex` in this file is a descriptor I created off of master before this was merged